src/v2/Apps/Artist/Components/ArtistSeriesRail/V2ArtistSeriesRail.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, BoxProps, Sans } from "@artsy/palette"
+import { Carousel } from "v2/Components/Carousel"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtistSeriesRail_artist } from "v2/__generated__/ArtistSeriesRail_artist.graphql"
+import { V2ArtistSeriesItemFragmentContainer as ArtistSeriesItem } from "./V2ArtistSeriesItem"
+import { ContextModule } from "@artsy/cohesion"
+
+interface Props extends BoxProps {
+  artist: ArtistSeriesRail_artist
+  title?: string
+  contextModule: ContextModule
+}
+
+/**
+ * TODO: When this page is migrated to V3:
+ * This component can be deleted and replaced with the V3 ArtistSeriesRail in
+ * v2/Components/ArtistSeriesRail
+ */
```

Comment:
This component was being used in a v2 context here on the artist page. So I copied the old one over into the artist directory.